src/executables/util_service.c

```diff
@@ -3950,50 +3950,13 @@ us_hb_process_start (HA_CONF * ha_conf, const char *db_name, bool check_result)
 
   print_message (stdout, MSGCAT_UTIL_GENERIC_START_STOP_2S, PRINT_HA_PROCS_NAME, PRINT_CMD_START);
 
-  pids = da_create (100, sizeof (int));
```

Comment:
It would be nice to remove together un-used local variables(i, pid, pids) and parameter variable (check_result).

Reply:
Done. Thanks